mobile/lib/features/common/avatar.dart

```diff
@@ -1,17 +1,26 @@
+import 'dart:io';
+
 import 'package:flutter/material.dart';
 
 class Avatar extends StatelessWidget {
-  const Avatar({Key key, this.photo, @required this.avatarSize})
+  const Avatar(
+      {Key key, this.photo, @required this.avatarSize, this.backgroundColor})
       : super(key: key);
 
   final String photo;
   final double avatarSize;
+  final Color backgroundColor;
 
   @override
   Widget build(BuildContext context) {
     return CircleAvatar(
       radius: avatarSize,
-      backgroundImage: photo != null ? NetworkImage(photo) : null,
+      backgroundColor: photo == null ? backgroundColor : null,
+      backgroundImage: photo != null
+          ? Uri.parse(photo).isAbsolute
+              ? NetworkImage(photo)
+              : Image.file(File(photo)).image
```

Comment:
Whats that?

Reply:
If the image is from API it will be downloaded, but there's a possibility that's a file selected by the user and it's not sent to the API yet (from local filepath).